examples/README.md

```diff
@@ -216,3 +221,34 @@ spec:
 
 If you access the pod you can check that your data is avaialable at
 `/var/lib/www/html`
+
+## Cluster ID based configuration
+
+Before creating a storage class that uses the option `clusterID` to refer to a
+Ceph cluster,
+
+**NOTE**: Substitute the output of `ceph fsid` instead of `<cluster-fsid>` in
+    the mentioned template YAML files, and also the Ceph admin ID and
+    credentials in their respective options. Further, update options like
+    `monitors` and `pools` in the respective YAML files to contain the
+    appropriate information.
+
+Create the following config maps and secrets
+
+* `kubectl create -f ./rbd/template-ceph-cluster-ID-provisioner-secret.yaml`
+* `kubectl create -f ./rbd/template-ceph-cluster-ID-publish-secret.yaml`
```

Comment:
why we need `template-ceph-cluster-ID-provisioner-secret.yaml` and `template-ceph-cluster-ID-publish-secret.yaml` 
content of both file looks same, cant we use single file?

Reply:
I thought I had covered a similar doubt in the proposal, but look to be not so.

If we can have separate Ceph secrets for the provisioner and publisher parts of the operation we need separate secrets for the same, if not we just need one.

Existing implementation and this PR uses the same secret in 2 places, the existing implementation does so because k8s separates the names and hence has to provide it twice, this implementation does so to retain the flexibility that the secrets can be different.

If Ceph secrets cannot be different (or need to be the same), then this is not required and should become a single secret. @rootfs @gman0 can you help answer if Ceph secrets can differ?

Reply:
These are now removed, and like stated elsewhere we are back to user/admin id/key.